test/framework/envs/universal/env.go

```diff
@@ -55,3 +56,14 @@ func RestoreState(bytes []byte) {
 	Expect(Cluster.AddNetworking(state.ZoneEgress, framework.Config.ZoneEgressApp)).To(Succeed())
 	Cluster.SetCp(cp)
 }
+
+func PrintCPLogsOnFailure(report ginkgo.Report) {
```

Comment:
we probably need a separate function because the cluster implem is different but theoretically could be done with one fn.